.gitignore

```
@@ -23,4 +23,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-.firebaserc
\ No newline at end of file
+.firebaserc
+settings.json 
```

This should be the path of your file. You'll know it is correct as you'll see the file beeing actually omited from your PR
```suggestion
.vscode/settings.json 
```